### PR TITLE
Waterfall

### PR DESCRIPTION
**Pull request by cmaulany**
waterfall is now contains mutliple blocks
<feature

### CODE REVIEW

**Comment by BdeBock** [comment]
Conflicts!

File: src/main/java/sem/group47/entity/WaterfallSpawner.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
+package sem.group47.entity;
+
+import java.awt.Graphics2D;
+import java.util.ArrayList;
+
+import sem.group47.tilemap.TileMap;
+
+public class WaterfallSpawner extends MapObject {
```
Comment (karinvangarderen) [inline src/main/java/sem/group47/entity/WaterfallSpawner.java]: I think the name waterfallSpawner is not very descriptive, if it just contains the waterfall throughout its lifetime. Perhaps waterfallComposite? WaterfallHolder?

**Comment by cmaulany** [comment]
is de highscore die dit steeds doet. is nu gefixt, en karin je hebt gelijk

**Comment by cmaulany** [comment]
maar nu werken de nieuwe powerups niet meer..

**Comment by BdeBock** [comment]
nice solution, will merge it.

**Pull request merged**